src/components/common/EthHashInfo/index.tsx

```diff
@@ -65,11 +71,13 @@ const EthHashInfo = ({
 
         <Box className={css.addressRow}>
           <Typography variant="body2" fontWeight="inherit" component="div" className={css.address}>
-            {prefix && <b>{prefix}:</b>}
-            {shortAddress ? shortenAddress(address) : address}
+            {showPrefix && shouldPrefix && prefix && <b>{prefix}:</b>}
+            {shortAddress && shortAddress ? shortenAddress(address) : address}
```

Comment:
Why this change?

Reply:
This was an oversight on my behalf. Good catch!